packages/components/src/components/page-error/page-error.jsx

```diff
@@ -1,30 +1,46 @@
+import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
 import ButtonLink from 'Components/button-link/button-link.jsx';
+import DesktopWrapper from 'Components/desktop-wrapper/desktop-wrapper.jsx';
 
-const PageError = ({ buttonOnClick, error_code_message, messages, header, redirect_label, redirect_url }) => (
-    <div className='dc-page-error__container'>
-        <div className='dc-page-error__box'>
+// if image_url is passed we should split the page to two columns and left-align messages
+const PageError = ({ buttonOnClick, messages, header, redirect_label, redirect_url, classNameImage, image_url }) => (
+    <div className={classNames('dc-page-error__container', { 'dc-page-error__container--left': !!image_url })}>
+        {!!image_url && (
+            <DesktopWrapper>
+                <img className={classNameImage} src={image_url} />
```

Comment:
I think we can add an `alt` tag as well (we could add `lazy` as well since it's supported by chrome now)?